src/views/auth/Signin.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <sbc-signin @sync-user-profile-ready="redirectOnLogin()" />
```

Comment:
When updating to 2.1.18 of the sbc common components on create business UI I had to change the listening event to keycloak-session-ready or I was experience a periodic refresh. No action required, just FYI.

Reply:
Thanks!